app/scripts/views/complete_sign_up.js

```diff
@@ -94,12 +96,14 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
               return false;
             }
 
-            return self.getAccount().isSignedIn()
+            var account = self.getAccount();
+            return account.isSignedIn()
               .then(function (isSignedIn) {
                 if (isSignedIn) {
                   self.navigate('settings', {
                     success: t('Account verified successfully')
                   });
+                  self._notifications.broadcast(self._notifications.EVENTS.SIGNED_IN, account.toJSON());
```

Comment:
[You mentioned this in the last PR](https://github.com/mozilla/fxa-content-server/pull/3206#discussion_r42737902) and I asked a question that never got answered, so I'll restate it here:

I thought the other condition meant that the user isn't being redirected to `/settings` and hence wasn't sure that we'd want to kick anything off in other tabs. Do we?